src/amo/components/Errors/AuthExpired/index.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+/* global window */
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import Button from 'amo/components/Button';
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import translate from 'amo/i18n/translate';
+import { logOutUser } from 'amo/reducers/users';
+import type { I18nType } from 'amo/types/i18n';
+import type { DispatchFunc } from 'amo/types/redux';
+
+type Props = {||};
+
+type DefaultProps = {|
+  _window: typeof window,
+|};
+
+type InternalProps = {|
+  ...Props,
+  ...DefaultProps,
+  dispatch: DispatchFunc,
+  i18n: I18nType,
+|};
+
+export class AuthExpiredBase extends React.Component<InternalProps> {
+  static defaultProps: DefaultProps = {
+    _window: typeof window !== 'undefined' ? window : {},
+  };
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(logOutUser());
+  }
+
+  render(): React.Node {
+    const { _window, i18n } = this.props;
+
+    const reloadButton = (
+      <Button buttonType="none" micro onClick={() => _window.location.reload()}>
```

Comment:
With `buttonType="none"`, `micro` doesn't do anything, so please remove it.

Reply:
The button is now rendered as text, which is great, but I think it should be coloured and underlined to highlight that it's a link. It just looks like the rest of the text, currently:

![Screen Shot 2022-01-19 at 1 35 40 PM](https://user-images.githubusercontent.com/142755/150193319-c38bab71-980a-4343-8e5a-e47df0acbc21.png)